database/scripts/add_multiple_known_errors.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative 'lib/buildfarm_tools'
+
+if ['-h', '--help', 'help'].include? ARGV.first
+  puts "Usage: '#{$0} <github_issue_url>'\n" \
+    'Write all the errors names in each line, ending with an empty newline'
+  exit(0)
+end
+
+github_issue = ARGV.first
+
+unless github_issue.include? 'https://github.com/'
+  puts "Specify github issue using '#{$0} <github_issue_url>'"
```

Comment:
Maybe a nit, but what if the issue is saved in gitlab? (I can think of `tracetools` that IIRC it's there)